lib-rca/src/main/java/com/facebook/openwifi/librca/models/ClientStats.java

```diff
@@ -0,0 +1,12 @@
+package com.facebook.openwifi.librca.models;
+
+import java.util.List;
+
+/** Statistic Model for each client. Maily handle KPI and metric calculations */
+public class ClientStats {
+    /** Unique ID for each client. */
+    String station;
+    
+    /** LinkStats that are of the same station(client). Because of disconnection issues, a client might be connected to several BSSIDs. */
```

Comment:
I think this javadoc is a bit misleading - it seems to imply that disconnection issues are why there may be multiple `LinkStats` objects. I think you could just leave it at `LinkStats that are of the same station(client)` and if someone wanted more info they could look at the javadoc for `LinkStats`.

The `LinkStats` are aggregated by bssid, station, and radio config. This means each `LinkStats` will have a unique bssid, station, and radio config. Therefore, if the radio config changes (e.g., different tx power), there will be multiple `LinkStats` created.

Reply:
I agree that we should remove "Because of disconnection issues,"